hw/system/occamy/src/occamy_cfg.hjson

```diff
@@ -168,10 +154,102 @@
         num_itlb_entries: 1,
     }
     // peripherals
+    debug: {
```

Comment:
If you want to add configuration items to the config (which is a good idea), please also remember to adapt the documentation schema (`docs/schema/occamy.schema.json`); otherwise, the documentation does not match the config, which is a bad time for people getting started.

You will then also have to rebuild the doc, which is a bit of a pain. You can adapt the schema and I will push a build commit.

Reply:
Ah completely forgot the schema, of course, I will add them! Ideally, we would have a CI check to ensure they are kept up-to-date.